frontend/pages/_error.tsx

```diff
@@ -27,6 +28,8 @@ const CustomErrorComponent: NextPage<ErrorProps> = (props) => {
 };
 
 CustomErrorComponent.getInitialProps = async (contextData) => {
+   contextData.res?.setHeader('Cache-Control', CACHE_CONTROL_DISABLED);
```

Comment:
It might be nice to updated `withCache` to be able to handle this case and just wrap this function.

Reply:
It would be nice to keep all the cache control logic in one place.

It might be tricky to reuse `withCache` here because it is designed to work with `getServerSideProps`, not `getInitialProps`. I didn't want to go down the rabbit hole of abstracting the `GetServerSidePropsMiddleware` types. But if you have any suggestions I'm all ears

Reply:
I was thinking it might be easy to make two separate functions but yea I took a very quick stab at it and agree this is easy for now.

Reply:
Hmm. If it's possible, it might be worth trying our best to consolidate the caching logic _as much as possible_. I suspect that keeping the consts for `no-cache, no-store`, et alia, near the caching middleware is a good breadcrumb. That'd be nice to let that module evolve, even if we can't use the same exact HoC.